### ISSUE TITLE
［FVT］:rflash <noderange> -c gives out unwanted message in python

### ISSUE THREAD

**Issue by tingtli** [xcat2/xcat-core#4765]
xCAT 2.13.10

rflash <noderange> -c gives out unwanted message. It gives out the firmware which is not in active status.

Here is the output of python version
```
[root@briggs01 firm]# rflash mid05tor12cn05 -c
mid05tor12cn05: BMC Firmware Product: ibm-v2.0-0-r26.1-0-gfb7714a (Active)+
mid05tor12cn05: BMC Firmware Product: ibm-v2.0-0-r26-0-g7285b52 (Active)*
mid05tor12cn05: Host Firmware Product: IBM-witherspoon-ibm-OP9_v1.19_1.90 (Active)*
mid05tor12cn05: Host Firmware Product: -- additional info: buildroot-2017.08-8-g5e23247
mid05tor12cn05: Host Firmware Product: -- additional info: capp-ucode-p9-dd2-v2
mid05tor12cn05: Host Firmware Product: -- additional info: hostboot-binaries-b4606e0
mid05tor12cn05: Host Firmware Product: -- additional info: hostboot-c646b78
mid05tor12cn05: Host Firmware Product: -- additional info: linux-4.13.11-openpower1-p5bc6a1c
mid05tor12cn05: Host Firmware Product: -- additional info: machine-xml-bcd4a40
mid05tor12cn05: Host Firmware Product: -- additional info: occ-51965bd
mid05tor12cn05: Host Firmware Product: -- additional info: op-build-v1.19-276-g91e7614
mid05tor12cn05: Host Firmware Product: -- additional info: petitboot-v1.6.3-pd4b114c
mid05tor12cn05: Host Firmware Product: -- additional info: sbe-6d7627a
mid05tor12cn05: Host Firmware Product: -- additional info: skiboot-v5.9.4
```
Here is the output of perl version
```
[root@briggs01 ~]# rflash mid05tor12cn05 -c
mid05tor12cn05: BMC Firmware Product:   ibm-v2.0-0-r26-0-g7285b52 (Active)*
mid05tor12cn05: HOST Firmware Product:   IBM-witherspoon-ibm-OP9_v1.19_1.90 (Active)*
mid05tor12cn05: HOST Firmware Product: -- additional info: buildroot-2017.08-8-g5e23247
mid05tor12cn05: HOST Firmware Product: -- additional info: capp-ucode-p9-dd2-v2
mid05tor12cn05: HOST Firmware Product: -- additional info: hostboot-binaries-b4606e0
mid05tor12cn05: HOST Firmware Product: -- additional info: hostboot-c646b78
mid05tor12cn05: HOST Firmware Product: -- additional info: linux-4.13.11-openpower1-p5bc6a1c
mid05tor12cn05: HOST Firmware Product: -- additional info: machine-xml-bcd4a40
mid05tor12cn05: HOST Firmware Product: -- additional info: occ-51965bd
mid05tor12cn05: HOST Firmware Product: -- additional info: op-build-v1.19-276-g91e7614
mid05tor12cn05: HOST Firmware Product: -- additional info: petitboot-v1.6.3-pd4b114c
mid05tor12cn05: HOST Firmware Product: -- additional info: sbe-6d7627a
mid05tor12cn05: HOST Firmware Product: -- additional info: skiboot-v5.9.4
```

**Comment by xuweibj** [comment]
Fixed in pull request #4969.

**Comment by tingtli** [comment]
Verified with xcat 2.14 3/26 build. Fixed.
```
[root@briggs01 ~]# lsxcatd -v
Version 2.14 (git commit 3217230d197babb3688fef28f26135fb5e5ad1a1, built Mon Mar 26 06:15:52 EDT 2018)

[root@briggs01 ~]# rflash mid05tor12cn05 -c
Tue Mar 27 04:47:49 2018 mid05tor12cn05: [openbmc_debug] login curl -k -c cjar -b cjar -X POST -H "Content-Type: application/json" https://172.11.139.5/login -d '{"data": ["root", "xxxxxx"]}'
Tue Mar 27 04:47:49 2018 mid05tor12cn05: [openbmc_debug] login 200 OK
Tue Mar 27 04:47:49 2018 mid05tor12cn05: [openbmc_debug] list_firmware curl -k -c cjar -b cjar -X GET -H "Content-Type: application/json" https://172.11.139.5/xyz/openbmc_project/software/enumerate
Tue Mar 27 04:47:50 2018 mid05tor12cn05: [openbmc_debug] list_firmware 200 OK
mid05tor12cn05: BMC Firmware Product: ibm-v2.0-0-r35-0-g7c7c1af (Active)*
mid05tor12cn05: Host Firmware Product: IBM-witherspoon-ibm-OP9_v1.19_1.125 (Active)*
mid05tor12cn05: Host Firmware Product: -- additional info: buildroot-2017.08-8-g5e23247
mid05tor12cn05: Host Firmware Product: -- additional info: capp-ucode-p9-dd2-v2
mid05tor12cn05: Host Firmware Product: -- additional info: hostboot-89d9dff
mid05tor12cn05: Host Firmware Product: -- additional info: hostboot-binaries-38248b4
mid05tor12cn05: Host Firmware Product: -- additional info: linux-4.13.16-openpower1-pb7e06ca
mid05tor12cn05: Host Firmware Product: -- additional info: machine-xml-758eb02
mid05tor12cn05: Host Firmware Product: -- additional info: occ-84f3564
mid05tor12cn05: Host Firmware Product: -- additional info: op-build-v1.19-333-g987ca39-dirty
mid05tor12cn05: Host Firmware Product: -- additional info: petitboot-v1.6.3-p6897366
mid05tor12cn05: Host Firmware Product: -- additional info: sbe-99e2fe2
mid05tor12cn05: Host Firmware Product: -- additional info: skiboot-v5.9-240-g081882690163
```
